rootston/config.c

```diff
@@ -240,6 +240,9 @@ static int config_ini_handler(void *user, const char *section, const char *name,
 		if (strcmp(name, "xwayland") == 0) {
 			if (strcasecmp(value, "true") == 0) {
 				config->xwayland = true;
+			} else if (strcasecmp(value, "lazy") == 0) {
+				config->xwayland = true;
```

Comment:
I don't think this should enable xwayland, because it's convenient to toggle xwayland just by setting `xwayland` to `false`

Reply:
It's the same option that is set to lazy instead of true/false, so I'm not sure your comment applies?
It could be a brand new option but I don't see what that would bring.

Reply:
Yes, it is the same option, that did feel nicer than making it a separate option.
So the options for Xwayland in the config are true, lazy and false.

Reply:
Right, I misunderstood the code, disregard my comment!